.github/workflows/publish-helm-chart.yaml

```diff
@@ -0,0 +1,174 @@
+---
+name: Publish Release
+on:
+  workflow_dispatch: {}
+  push:
+    branches: [ master ]
+    tags-ignore: [ "*-dev+[0-9]+-g*" ]
+jobs:
+  gather-info:
+    name: Gather Release Information
+    runs-on: ubuntu-latest
+    outputs:
+      github-ref: ${{ steps.github-ref.outputs.github-ref }}
+      docker-tag: ${{ steps.docker-tag.outputs.docker-tag }}
+      git-tag: ${{ steps.git-tag.outputs.git-tag }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - id: github-ref
+      name: Determine GitHub Ref
+      env:
+        GITHUB_REF: ${{ github.ref }}
+      run: |
+        case "${GITHUB_REF}" in
+          refs/tags/*)
+            GITHUB_REF="${GITHUB_REF#refs/tags/}" ;;
+          refs/pull/*)
+            GITHUB_REF="${GITHUB_REF#refs/pull/}"
+            GITHBU_REF="pr-${GITHUB_REF//\//-}" ;;
+          refs/heads/*)
+            GITHUB_REF="${GITHUB_REF#refs/heads/}"
+            GITHUB_REF="${GITHUB_REF//\//-}" ;;
+          *)
+            echo "Unexpected GITHUB_REF ${GITHUB_REF}" >&2
+            exit 1 ;;
+        esac
+        echo "GitHub ref found: ${GITHUB_REF}"
+        echo "::set-output name=github-ref::${GITHUB_REF}"
+    - id: git-tag
+      name: Determine Git Tag
```

Comment:
I assume that's https://github.com/cloudfoundry-incubator/kubecf-tools/tree/main/versioning ? Is there a readme on what the recommended way to actually use things from that repo is?

Reply:
You can use kubecf as an example [1].
This versioning should only be applied to the docker image now.
We don't have to build another chart (since this is only usable by kubecf), but include the templates directly in kubecf.

https://github.com/cloudfoundry-incubator/kubecf/blob/master/.gitmodules#L4